From 6ca9ae70bb9f09b6bc5d9f26e0a710a8691199e3 Mon Sep 17 00:00:00 2001
From: Arkadiusz Hiler <arek@hiler.eu>
Date: Tue, 13 Oct 2020 18:20:39 +0300
Subject: [PATCH] HACK: wine.inf: Enable atiadlxx and disable ags for Shadow of
 War

With the recent stubs game is able to use our atiadlxx without crashing.
This helps with getting the resolution list populated with AMD GPUs.

AGS has to be force disabled - the dll ships with the game and uses a
bunch of more complex ADL2 calls that remain unimplemented. The game
works fine if the DLL is not found.
---
 loader/wine.inf.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 8c1742a95e7..e2efcce2a31 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -4021,6 +4021,8 @@ HKCU,Software\Wine\AppDefaults\s2_sp64_ship.exe\DllOverrides,"atiadlxx",,"builti
 HKCU,Software\Wine\AppDefaults\h1_sp64_ship.exe\DllOverrides,"atiadlxx",,"builtin"
 HKCU,Software\Wine\AppDefaults\iw7_ship.exe\DllOverrides,"atiadlxx",,"builtin"
 HKCU,Software\Wine\AppDefaults\BlackOps3.exe\DllOverrides,"atiadlxx",,"builtin"
+HKCU,Software\Wine\AppDefaults\ShadowOfWar.exe\DllOverrides,"atiadlxx",,"builtin"
+HKCU,Software\Wine\AppDefaults\ShadowOfWar.exe\DllOverrides,"amd_ags_x64",,""
 ;;Likely want *80 and *90 too, but those require removing Wine's manifest files.
 HKCU,Software\Wine\DllOverrides,"atl100",0x2,"native,builtin"
 HKCU,Software\Wine\DllOverrides,"msvcp100",0x2,"native,builtin"
